javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -344,30 +344,22 @@ public void onTransformation(
         ClassLoader classLoader,
         JavaModule module,
         boolean loaded,
-        DynamicType dynamicType) {
-      log.debug("Transformed {} -- {}", typeDescription.getName(), classLoader);
-    }
+        DynamicType dynamicType) {}
```

Comment:
Do you think we'll later add something that adds log messages to a list or something to print them? Guess `Transformed` is a pretty useful message for debugging instrumentation load.

Reply:
hm, you're right, this is a pretty useful debug log

that's not a bad idea, under a system property, we could write to a queue, and have another thread log it (so the logging won't happen inside of the class file transform)

will give this PR another try 👍

Reply:
Talked offline, let's follow up so our current builds (especially release build) are more stable